metaphor/redshift/lineage/extractor.py

```diff
@@ -147,9 +147,7 @@ async def _fetch_lineage(self, sql, conn, db):
             source_table_name = f"{db}.{row['source_schema']}.{row['source_table']}"
             target_table_name = f"{db}.{row['target_schema']}.{row['target_table']}"
             query = (
-                row["querytxt"].rstrip()
-                if row.get("querytxt") is not None
-                else None
+                row["querytxt"].rstrip() if row.get("querytxt") is not None else None
```

Comment:
Oops sorry shouldn't have suggested double negation

```suggestion
                None if row.get("querytxt") is None else row["querytxt"].rstrip()
```

Reply:
Your suggestion was actually correct. I also updated test cases.